### ISSUE TITLE
CC0 internationally 

### ISSUE THREAD

**Issue by mpfl** [mozillascience/checklist#3]
For this to be a truly international checklist, you might need to rethink the recommendation of CC0. It might not be a legal licence in Australia:

http://www.ausgoal.gov.au/research-data-faqs#Q4

**Comment by stephwright** [comment]
Ahh... thanks @mpfl!  I wasn't aware of the Australian issue w/ CC0.  I've started down a rabbit hole of int'l law on CC0/public domain.  I'll leave this issue open while I dig into it further.

**Comment by mpfl** [comment]
I'm sorry about that! Getting familiar with one country's IP laws is hard enough. International law is the route to madness!

**Comment by stephwright** [comment]
No, no, no!  Don't apologize!  That's why I put out the call for feedback.  I appreciate knowing this! Now if only I'd gone for that law degree instead of the librarian degree so many years ago. ;)
